### PR TITLE
add Daosheng's slides

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by ngokevin** [comment]
I'm don't think the slides are very useful as a learning resource on their own. It'd be nice if we had a video of the full talk.

**Comment by cvan** [comment]
> I don't think the slides are very useful as a learning resource on their own. It'd be nice if we had a video of the full talk.

perhaps I could wait until they have that. the slides actually seemed quite good IMO.

File: README.md [hunk 1]
```diff
@@ -38,11 +38,12 @@ Learn how to use A-Frame.
 - [Hands-on With Virtual Reality Using A-Frame, React and Redux](https://medium.com/immersion-for-the-win/hands-on-with-virtual-reality-using-a-frame-react-and-redux-bc66240834f7#.nce7sfdzl)
 - [Schneller Einstieg in die WebVR Entwicklung mit A-Frame](http://thevrjump.com/de/aframe-einstieg) (German) - [Translated](https://translate.google.com/translate?hl=en&sl=auto&tl=en&u=http%3A%2F%2Fthevrjump.com%2Fde%2Faframe-einstieg)
 - [WebVR mit A-Frame: Einführung & Grundlagen](http://thevrjump.com/de/aframe-grundlagen) (German) - [Translated](https://translate.google.com/translate?hl=en&sl=auto&tl=en&u=http%3A%2F%2Fthevrjump.com%2Fde%2Faframe-grundlagen)
+- [Introduction to A-Frame](http://www.slideshare.net/ellisonmu/introduction-to-aframe-57170744) - slides from talk at Mozilla Taipei (by [@daoshengmu])
```
Comment (ngokevin) [inline README.md]: Capitalize description

**Comment by ngokevin** [comment]
r+wc anyways, we can get the talk in later

File: README.md (hunk 1 above)
Comment (cvan) [inline README.md]: that's fine

**Pull request merged**